src/api/createVoiceVideoCalling.ts

```diff
@@ -239,7 +239,6 @@ export class VoiceVideoCallingProxy {
             });
 
             console.error(`[VoiceVideoCallingProxy][rejectCall][makeSecondaryChannelEventRequest] Failure ${e}`);
-            return e;
```

Comment:
Why delete this?

Reply:
TS was not happy with the fact that the method is returning `Promise<void>` & an error object is returned.